examples/postgres/getting_started_step_1/src/bin/show_posts.rs

```diff
@@ -5,11 +5,11 @@ use diesel_demo_step_1_pg::*;
 fn main() {
     use self::schema::posts::dsl::*;
 
-    let connection = establish_connection();
+    let mut connection = establish_connection();
     let results = posts
         .filter(published.eq(true))
         .limit(5)
-        .load::<Post>(&connection)
+        .load::<Post>(&mut connection)
         .expect("Error loading posts");
```

Comment:
Oh and last idea:
It feels a bit boilerplate-ish to have to borrow the connection mutably everytime, so I was wondering whether we'd like to change the suggested way to connect (and the way we use it in our tests) to:
```rust
    let connection = &mut establish_connection();
    let results = posts
        .filter(published.eq(true))
        .limit(5)
        .load::<Post>(connection)
        .expect("Error loading posts");
```

(note: `let connection = &mut establish_connection();` is equivalent to:
```rust
    let mut connection = establish_connection();
    let connection = &mut connection;
```
)